src/ImageSharp/Colors/PackedPixel/IPixel.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ImageSharp
+{
+    using System;
+    using System.Numerics;
+
+    public interface IPixel
```

Comment:
The generic version does not seem to provide any kind of useful polymorphic behaviour to me. Looks like a legacy concept even in monogame. Having interfaces only to enforce signature (not for the sake of polymorphism) ...  not my favorite stuff.